examples/grpc-bridge/README.md

```diff
@@ -0,0 +1,31 @@
+# envoy-grpc
+
+This is an example usage of the Envoy [gRPC bridge filter](https://lyft.github.io/envoy/docs/configuration/http_filters/grpc_http1_bridge_filter.html#config-http-filters-grpc-bridge). Included is a gRPC in memory Key/Value store with a Python HTTP client. The Python client makes HTTP/1 requests through the Envoy sidecar process which are upgraded into HTTP/2 gRPC requests. Response trailers are then buffered and sent back to the client as a HTTP/1 header payload.
+
```

Comment:
To kill two birds with one stone we could also mention how we are doing authority based routing, which differs from how routing is done in the front-proxy example.